routes/users.js

```diff
@@ -0,0 +1,80 @@
+import express from 'express';
+import userSchema from '../schema/user.js'
+import requestSchema from '../schema/request-query.js'
+import {mapErrors, validateSchema} from "../utils.js";
+import {v4 as uuid} from 'uuid';
+import bcrypt from 'bcrypt';
+import dbClient from '../db/db-client.js';
+
+const router = express.Router();
+const saltRounds = 10;
+
+router.get('/', validateSchema(requestSchema, 'query'), (req, res) => {
+    const usersColl = dbClient.collection('users');
+    const users = usersColl.getAll({...req.query, searchBy: 'login'});
+    res.json(users);
+}, (err, req, res, next) => {
+    const errors = [{message: 'Can not get users: ' + err}];
+    res.status(500).json(mapErrors(errors));
+});
+
+router.get('/:id', (req, res) => {
```

Comment:
please, take a took at `param` method http://expressjs.com/en/4x/api.html#router.METHOD for optimizing repeating operations over the record with proper `:id`